clang/test/Driver/Ofast.c

```diff
@@ -10,6 +10,7 @@
 // RUN: %clang -Ofast -fno-strict-aliasing -### %s 2>&1 | FileCheck -check-prefix=CHECK-OFAST-NO-STRICT-ALIASING %s
 // RUN: %clang -Ofast -fno-vectorize -### %s 2>&1 | FileCheck -check-prefix=CHECK-OFAST-NO-VECTORIZE %s
 
+// CHECK-OFAST: warning: argument '-Ofast' is deprecated; use '-O3', possibly with '-ffast-math'
```

Comment:
Do we have a test that for `-Ofast -O3` ?

Reply:
We have `RUN: %clang -Ofast -O2` in this file, which does validate that `-O2` overrides `-Ofast`. However, it should have a CHECK-NOT to ensure it also doesn't emit a diagnostic.

Reply:
Added `CHECK-NOT` test.